gprofiler/profilers/node.py

```diff
@@ -127,39 +139,54 @@ def _execute_js_command(sock: WebSocket, command: str) -> Any:
     except json.JSONDecodeError:
         raise NodeDebuggerUnexpectedResponse(message) from None
     try:
-        return message["result"]["result"]["value"]
+        if isinstance(message["result"]["result"]["value"], str) and "process undefined" in message["result"]["result"]["value"]:
```

Comment:
```suggestion
        if isinstance(message["result"]["result"]["value"], str) and "process undefined" == message["result"]["result"]["value"]:
```

Shouldn't it be equal?